cluster/manager.go

```diff
@@ -248,6 +250,20 @@ func (c *ClusterManager) getNonDecommisionedPeers(db ClusterInfo) map[types.Node
 	return peers
 }
 
+func (c *ClusterManager) getBootstrapPeers(dci *discovery.ClusterInfo) map[types.NodeId]string {
+	peers := make(map[types.NodeId]string)
+	for _, bne := range dci.Nodes {
+		peers[types.NodeId(bne.Id)] = bne.Ip + ":9002"
```

Comment:
nit: "9002" can be a constant. Not sure why you need to add the constant - seems like if all of them have same port then we probably don't need to add it at all .. ?